editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.spec.browser2.tsx

```diff
@@ -364,6 +407,462 @@ describe('Absolute Resize Strategy', () => {
       `),
     )
   })
+  it('resizes absolute element with aspect ratio locked, keeping aspect ratio', async () => {
+    const renderResult = await renderTestEditorWithCode(
+      projectWithAspectRatioLockedElement,
+      'await-first-dom-report',
+    )
+
+    const target = EP.elementPath([['0cd', '7a0']])
```

Comment:
if an element has data-uid prop, it means you can give it an arbitrary uid (so long as it's unique), so you could make this selection ['aaa', 'bbb'], or something similar human readable, like ['sb', 'aspect-ratio-locked-element']

Reply:
thanks! fixed

Reply:
thanks! updated the uid